src/main/java/io/nats/client/impl/MessageQueue.java

```diff
@@ -47,11 +45,12 @@ class MessageQueue {
     protected final long offerTimeoutNanos;
     protected final Duration requestCleanupInterval;
 
-    // Poison pill is a graphic, but common term for an item that breaks loops or stop something.
+    // SPECIAL MARKER MESSAGES
+    // A simple == is used to resolve if any message is exactly the static pill object in question
+    // ----------
+    // 1. Poison pill is a graphic, but common term for an item that breaks loops or stop something.
     // In this class the poison pill is used to break out of timed waits on the blocking queue.
-    // A simple == is used to check if any message in the queue is this message.
-    // /\ /\ /\ /\ which is why it is now a static. It's just a marker anyway.
-    protected static final NatsMessage POISON_PILL = new NatsMessage(POISON, null, EMPTY_BODY);
+    protected static final NatsMessage POISON_PILL = new NatsMessage("_poison", null, EMPTY_BODY);

```

Comment:
I was in this file looking at the marker pill and ended up doing some refactoring. No functionality was changed and I clarified the comment.